### PR TITLE
Add node resizin

### LINKED ISSUE #3: Add node-resizin package
Add `node-resizin` subpackage which replace `ackke-node-sdk-image-server` and reuse most of the upload funcionality.

The package should polyfill node environment with form data, take an inspiration here
https://github.com/form-data/isomorphic-form-data/blob/master/lib/index.js

### PR DESCRIPTION
**Pull request by horaklukas**
Fixes #3 

Add `node-resizin` package, that reuse `resizin` package and uniforms the interface. The package only polutes global environment with `FormData` so that `resizin` package doesnt need to carry about it.

### CODE REVIEW

File: packages/node-resizin/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { uploadFactory } from 'resizin';
+import FormData from 'form-data';
+
+(global as any).FormData = FormData;
```
Comment (baierjak) [inline packages/node-resizin/src/index.ts]: Whats this syntax?

File: packages/resizin/src/upload/upload.ts [hunk 2]
```diff
@@ -11,35 +34,40 @@ const uploadImage = (serverUrl: string, apiKey: string, imageId: string, file: s
...
         };
 
-        // TODO - polyfill FormData in node-resizing package
-        let formData;
-        if (typeof window === 'undefined') {
-            formData = new NodeFormData();
-        } else {
-            formData = new FormData();
-        }
+        const formData = new FormData();
```
Comment (baierjak) [inline packages/resizin/src/upload/upload.ts]: this FormData are the global one transformed above in node part?

File: packages/resizin/src/upload/uploadFactory.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import upload, { FileType } from './upload';
+import { DEFAULT_API_URL } from '../constants';
+
+interface ClientOptions {
+    serverUrl?: string;
+    apiKey: string;
+    fileType?: FileType;
+}
+
+const uploadFactory = (options: ClientOptions = {} as ClientOptions) => (
```
Comment (baierjak) [inline packages/resizin/src/upload/uploadFactory.ts]: ClientOptions = {} as ClientOptions? Is this correct?

File: packages/node-resizin/src/index.ts (hunk 1 above)
Comment (horaklukas) [reply packages/node-resizin/src/index.ts]: global namespace (which is node global of course) has a type definition which does not contain the `FormData` class. To be able to put it on it, first we need to extend type of global namespace. But you're right, I'm going to adjust it to make it more obvious.

File: packages/resizin/src/upload/upload.ts (hunk 2 above)
Comment (horaklukas) [reply packages/resizin/src/upload/upload.ts]: Depends on if it's used as a standalone package `@ackee/resizin` or through a node package `@ackee/node-resizin`.

File: packages/resizin/src/upload/uploadFactory.ts (hunk 3 above)
Comment (horaklukas) [reply packages/resizin/src/upload/uploadFactory.ts]: Syntax is correct, but maybe it could be written better way, I'll look at it.

**Pull request merged**